packages/sdks/javascript/README.md

```diff
@@ -74,25 +74,59 @@ getUserProfile();
 This package includes TypeScript declarations for every method.
 TypeScript versions `>= 3.8` are supported.
 
-Some methods in `Devopness SDK JavaScript` accept and return objects from the Devopness API. The type declarations for these objects will always track the latest version of the API. Therefore, if you'e using the latest version of this package, you can rely on the Devopness API documentation for checking the input and return types of each API endpoint.
+**Some methods in `Devopness SDK JavaScript` accept and return objects from the Devopness API. The type declarations for these objects will always track the latest version of the API. Therefore, if you'e using the latest version of this package, you can rely on the Devopness API documentation for checking the input and return types of each API endpoint.**
 
 ## Building and testing
 To build and test the SDK locally, follow these steps:
-1. Clone de repository
+
+## 1. Clone de repository
```

Comment:
This repo should be forked and not cloned.

Reply:
@jfoliveira resolved